src/carbon_calculator/CCDefaults.py

```diff
@@ -120,6 +122,7 @@ def exportDefaults(self,fileName):
             csvfile.close()
         return status
     def importDefaults(self,fileName):
+        csvfile = None
```

Comment:
when something goes wrong in the function, the finally clause uses the csvfile variable which is not declared.  I am initializing it here so that the code does not break